docs/versions.md

```diff
@@ -0,0 +1,19 @@
+EarlGrey Releases and their corresponding EarlGrey Gem Versions. The provided gem version is **not**
+the last compatible EarlGrey gem version for a release, but the gem version that was either released with the CocoaPod or
+with which the CocoaPod was tested for release.
```

Comment:
How about we say it's the recommended version? older or newer versions may be compatible but they've not been tested together.